conjure-java-core/src/integrationInput/java/com/palantir/product/EnumExample.java

```diff
@@ -28,6 +31,9 @@
     /** Value of 100. */
     public static final EnumExample ONE_HUNDRED = new EnumExample(Value.ONE_HUNDRED, "ONE_HUNDRED");
 
+    public static final List<EnumExample> VALUES =
+            Collections.unmodifiableList(Arrays.asList(ONE, TWO, ONE_HUNDRED));
```

Comment:
this doesn't actually encompass all the possible values as it's possible for `EnumExample`, for instance, to contain an infinite number of other potential, unknown values.

Reply:
For most use cases that I can think of, I don't particularly care about any potential unknown values, since there won't really be anything I can do with them anyway and I'll be using the method on the server side, where I think we generally expect to have the newer version of a given type.

Reply:
What about we rename this from `values` to `knownValues` so that we lower the expectations?